src/graph/visitor/VidExtractVisitor.cpp

```diff
@@ -194,10 +207,9 @@ void VidExtractVisitor::visit(RelationalExpression *expr) {
     vidPattern_ = VidPattern{VidPattern::Special::kInUsed,
                              {{fCallExpr->args()->args().front()->toString(),
                                {VidPattern::Vids::Kind::kIn, List({constExpr->value()})}}}};
-    return;
   } else {
-    vidPattern_ = VidPattern{};
-    return;
+    vidPattern_ = VidPattern{VidPattern::Special::kInUsed,
```

Comment:
I think you should check left expression is property related expression like `v.tag.prop > 3 or id(v) == 1`